Description: opendkim/opendkim.c:dkimf_config_free(): don't assert conf->refcnt == 0
Bug: https://github.com/trusteddomainproject/OpenDKIM/issues/22

--- a/opendkim/opendkim.c
+++ b/opendkim/opendkim.c
@@ -5902,7 +5902,6 @@
 dkimf_config_free(struct dkimf_config *conf)
 {
 	assert(conf != NULL);
-	assert(conf->conf_refcnt == 0);
 
 	dkimf_zapkey(conf);
 
